packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
-
+-   [**] Enable embed inline previews (for now only YouTube and Twitter will be available) [#34447]
```

Comment:
I think the PR should be #34446 instead of #34447 now 😅

Reply:
Oh good catch 😅 , I merged the changes but I forgot to update this entry, I'll update it asap.